package.json

```diff
@@ -22,7 +22,8 @@
     "wo-imap-handler": "~0.1.11",
     "mimefuncs": "~0.3.4",
     "axe-logger": "~0.0.2",
-    "tcp-socket": "~0.4.0"
+    "tcp-socket": "~0.4.0",
+    "es6-promise": "^2.0.1"
```

Comment:
I followed the lead of [openpgp.js](https://github.com/openpgpjs/openpgpjs/blob/39e1e145e40cd7bbbf89daf4f967b1181d4873f1/package.json#L49) on this. I guess the alternative would be to move it to devDependencies and call the `ES6Promise.polyfill();` in the test setup not in the code and assume that Promise object is always available.